### ISSUE TITLE
GetMyFeesEstimateForASIN requests malformed

### ISSUE THREAD

**Issue by bartuszekj** [abuzuhri/Amazon-SP-API-CSharp#25]
GetMyFeesEstimateForASIN calls result in:

```
Amazon Api didn't respond with Okay, see exception for more details{
  "errors": [
    {
      "code": "InvalidInput",
      "message": "This is an invalid input",
      "details": "Missing objects [FeesEstimateRequest] in body parameters."
    }
  ]
}
```

Sample code:

```
	var id = Guid.NewGuid();
	var fer = new FeesEstimateRequest(MarketPlace.US.ID, true, new PriceToEstimateFees(new MoneyType("USD", 1469)), id.ToString());
	var feeEstimate = conn.ProductFee.GetMyFeesEstimateForASIN("0140440577", fer);
```

**Comment by bartuszekj** [comment]
Note: after doing some digging, it looks like changing this code in the library will work:

```
        public FeesEstimateResult GetMyFeesEstimateForASIN(string ASIN, FeesEstimateRequest feesEstimateRequest)
        {

            CreateAuthorizedRequest(ProductFeeApiUrls.GetMyFeesEstimateForASIN(ASIN), RestSharp.Method.POST, **postJsonObj: new { FeesEstimateRequest = feesEstimateRequest }**);
            var response = ExecuteRequest<GetMyFeesEstimateResponse>();
            if (response != null && response.Payload != null)
                return response.Payload.FeesEstimateResult;
            return null;
        }
```

**Comment by abuzuhri** [comment]
Hi @bartuszekj ,

Thanks for report that bug , its fixed and ready in nuget v1.0.18
